From 7d3fa6aa6f934b5ee21b9bfc4bcb242b8a522a4e Mon Sep 17 00:00:00 2001
From: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
Date: Wed, 15 Sep 2021 17:29:25 +0300
Subject: [PATCH 003/322] [FIO temphack] colibri_imx7_emmc: fix linking mxsfb
 driver

video_link library is required for mxsfb driver. Since
CONFIG_VIDEO_MXS is not converted to Kconfig, VIDEO_LINK can't be
selected automatically. Select the option VIDEO_LINK manually to fix
linkage error [1].

[1]
drivers/built-in.o: in function `mxs_of_get_timings':
drivers/video/mxsfb.c:487: undefined reference to `video_link_get_next_device'
drivers/video/mxsfb.c:489: undefined reference to `video_link_get_display_timings'

Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---
 configs/colibri_imx7_emmc_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/colibri_imx7_emmc_defconfig b/configs/colibri_imx7_emmc_defconfig
index c5a6c4433d5..f2943491e9d 100644
--- a/configs/colibri_imx7_emmc_defconfig
+++ b/configs/colibri_imx7_emmc_defconfig
@@ -94,4 +94,5 @@ CONFIG_SPLASH_SCREEN=y
 CONFIG_SPLASH_SCREEN_ALIGN=y
 CONFIG_VIDEO_BMP_RLE8=y
 CONFIG_BMP_16BPP=y
+CONFIG_VIDEO_LINK=y
 CONFIG_OF_LIBFDT_OVERLAY=y
-- 
2.34.1

